wgpu-core/platform-features/windows-linux-android-emscripten/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "wgpu-core-platform-windows-linux-android-emscripten"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+description = "Feature unification helper crate for the Windows/Linux/Android platforms"
+homepage.workspace = true
+repository.workspace = true
+keywords.workspace = true
+license.workspace = true
```

Comment:
for all proxy crates: should have a readme so an explanation for this snafu shows up on cratesio